Userland/Applications/PixelPaint/Guide.h

```diff
@@ -14,6 +14,7 @@ namespace PixelPaint {
 class Guide : public RefCounted<Guide> {
 public:
     enum class Orientation {
+        Unset,
```

Comment:
What does this value do, exactly?

Reply:
It's passed to the EditGuideDialog as Orientation when creating a new Guide to not have one of the radio buttons pre-selected

Reply:
Hmm...okay. I guess that's fine.

Reply:
Perhaps passing in an `Optional<Orientation>` would be better?. We can already compare `Optional<Orientation>` with `Orientation` types directly, so nothing else would really need to change, and you wouldn't need to have a separate `Unset` orientation value